### PR TITLE
Add millipad keypad and default keymap

### PR DESCRIPTION
**Pull request by GLozares**
## Description

Added qmk support for my custom keypad "millipad". This keypad is open source and is available on my github. There is a private group buy going on between my friends and I, but it is still open source.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ x ] Keyboard (addition or update)
- [ x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/millipad/readme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+![PCB Layout](https://i.imgur.com/ilhfBfQ.png)
+![Front of PCB](https://i.imgur.com/UiQkmuv.jpg)
+![Back of PCB](https://i.imgur.com/Inh6UTL.jpg)
+
+
+* Keyboard Maintainer: [Jirou](https://github.com/GLozares)
+* Hardware Supported: Jirou.Design Millipad
+* Hardware Availability: Open Source at https://github.com/GLozares/millipad
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/millipad/readme.md:20]: Could you add instructions on how to enter the bootloader to flash new firmware?

Comment (GLozares) [reply keyboards/millipad/readme.md:21]: I moved the location of the bootloader instructions to be closer to the make example and made it clearer in my latest commit.

**Review by drashna — APPROVED** [review]

File: keyboards/millipad/config.h [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+/* Bootmagic Lite key configuration */
+//#define BOOTMAGIC_LITE_ROW 0
+//#define BOOTMAGIC_LITE_COLUMN 0
+
+/* Encoder Definitions */
+
+#define ENCODERS_PAD_A { D3 }
+#define ENCODERS_PAD_B { D2 }
+#define ENCODER_RESOLUTION 3
```
Comment (fauxpark) [inline keyboards/millipad/config.h:94]: This is usually either 2 or 4.

File: keyboards/millipad/readme.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# millipad
+
+![millipad](https://i.imgur.com/ZM2qel5.jpg)
```
Comment (fauxpark) [inline keyboards/millipad/readme.md:3]:
```suggestion
![millipad](https://i.imgur.com/ZM2qel5h.jpg)
```

File: keyboards/millipad/readme.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+An open source 2x6 macropad with rotary encoder. Designed to be cheap and simple.
+
+
+## More Pictures:
+![PCB Layout](https://i.imgur.com/ilhfBfQ.png)
+![Front of PCB](https://i.imgur.com/UiQkmuv.jpg)
```
Comment (fauxpark) [inline keyboards/millipad/readme.md:10]:
```suggestion
![Front of PCB](https://i.imgur.com/UiQkmuvh.jpg)
```

File: keyboards/millipad/readme.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+![Back of PCB](https://i.imgur.com/Inh6UTL.jpg)
```
Comment (fauxpark) [inline keyboards/millipad/readme.md:11]:
```suggestion
![Back of PCB](https://i.imgur.com/Inh6UTLh.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by GLozares** [comment]
@fauxpark Committed new readme to reflect your changes requested

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**